stories/how-do-you-feel.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback, useState } from 'react'
+import { HowdoyoufeelButton, Howdoyoufeel } from '../app/components/how-do-you-feel'
+
+export default {
+    title: 'how-do-you-feel',
+    component: Howdoyoufeel,
+    decorators: [onDoneDecorator],
+}
+const Template = Component => args => <Component {...args} />
+
+
+function onDoneDecorator(Story, context) {
```

Comment:
Please see pair-compare.stories.jsx for an example of how to import the decorator from common, and check the result of onDone when you add {valid, value}. 

If you haven't done so, you many need to git merge origin/master to pull in all the latest stuff.